src/main/java/com/notably/logic/commands/DeleteCommand.java

```diff
@@ -34,13 +33,9 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         List<Person> lastShownList = model.getFilteredPersonList();
 
-        if (targetIndex.getZeroBased() >= lastShownList.size()) {
-            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
-        }
-
         Person personToDelete = lastShownList.get(targetIndex.getZeroBased());
         model.deletePerson(personToDelete);
-        return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, personToDelete));
```

Comment:
Rather than returning `null` (which might make the app buggy, if `null` is not expected by the callee), how about returning a CommandResult, but with some custom string as the first parameter?

Reply:
Let's refer to this comment as (2)